recipes/mofchecker/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mofchecker" %}
+{% set version = "0.6.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mofchecker-{{ version }}.tar.gz
+  sha256: 6c9154d8df22436d3e95594f07deef06f99bbc3d2360f466bda71199e485532f
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click ==7.*
+    - networkx ==2.*
```

Comment:
Is there any reason these packages are pinned more tightly than [upstream](https://github.com/kjappelbaum/mofchecker/blob/master/requirements.txt).

Reply:
Now aligned the version pins with the `requirements.txt`